packages/integration-tests/src/views/common.js

```diff
@@ -69,4 +72,7 @@ export const commonActions = {
       commonActions.clickNextButton();
     }
   },
+  openNewClusterPage: () => {
```

Comment:
I would suggest calling this function `visitNewClusterPage`, the verb `open` might suggest that it would be possible to open the page in different forms (pop-up, new tab, new window, etc..., btw all these are anti-patterns in Cypress) while all we want to allow is just simulate browsing (changing) to a different page